packages/medusa/src/strategies/__tests__/tax-calculation.js

```diff
@@ -170,17 +273,21 @@ describe("TaxCalculationStrategy", () => {
       "%s",
       async (title, { items, taxLines, context, expected, flags }) => {
         if (flags) {
-          Object.entries(flags).forEach(([key, value]) => featureFlagRouter.setFlag(key, value))
+          Object.entries(flags).forEach(([key, value]) =>
```

Comment:
comment: you will be able to clean that part after this pr is merged https://github.com/medusajs/medusa/pull/2136

Reply:
in the last version of the base branch, this part should not be there. You should get the following:
```ts
describe("TaxCalculationStrategy", () => {
  describe("calculate", () => {
    test.each(toTest)(
      "%s",
      async (title, { items, taxLines, context, expected, flags }) => {
        const featureFlagRouter = new FlagRouter(flags ?? {})
        const calcStrat = new TaxCalculationStrategy({
          featureFlagRouter
        })

        const val = await calcStrat.calculate(items, taxLines, context)
        expect(val).toEqual(expected)
      }
    )
  })
})
```